content/pages/2024/tickets.md

```diff
@@ -72,5 +72,20 @@ attend EuroSciPy:
 
 <br>
 
-### Ticket prices
-TBA
+
+
+### Get your ticket
+
+<html>
+  <head>
+    <script src="https://js.tito.io/v2" async></script>
+  </head>
+  <body>
+    <tito-widget
+      event="euroscipy/2024"
+    ></tito-widget>
+  </body>
+</html>
```

Comment:
```suggestion
</html>

**Booking after August 1st, ticket prices will increase as follow:**

|                  | Student (*) | Academic/Individual | Business |
|------------------|-------------|---------------------|----------|
| Tutorial only    | 100 €       | 200 €               | 350 €    |
| Conference only  | 100 €       | 200 €               | 350 €    |
| Combined ticket  | 150 €       | 300 €               | 550 €    |

In case that you have additional queries, please contact us at: <a href="mailto:info@euroscipy.org">info@euroscipy.org</a>.
```